KAS-3814: Merge app themis export

sergiofenoll opened the pull request:
Removes all usages of kaleidosQuery and replaces them with queries to virtuoso (as they are now one and the same).

I checked that all updates to virtuoso are compatible with Kaleidos. They should be. The only place we're writing overlapping data is in the public graph, some export metdata goes there. Given that the exported data is public anyway, I don't think this is an issue. Other stuff is job related data and those go in temp graphs.

sergiofenoll committed:
feat: replace queries to Kaleidos DB with queries to own DB

Kaleidos DB and local DB are the same now.

erikap reviewed (commented):
When the export services runs in the Kaleidos stack, the polling for new publication activities can be replaced with acting on delta's about `ext:ThemisPublicationActivity`.  Maybe better to change in a future story/PR.

sergiofenoll commented:
> When the export services runs in the Kaleidos stack, the polling for new publication activities can be replaced with acting on delta's about `ext:ThemisPublicationActivity`. Maybe better to change in a future story/PR.

:+1: Makes sense, but didn't want to make too many changes to the service in this story especially given the pressure with the other required stories this sprint. We need to revisit this anyway sometime soon because some form of monitoring is also necessary, so we can look at it later.

sergiofenoll committed:
feat: retry failed PublicExportJobs

Before we "silently" failed exports if e.g. Virtuoso's checkpointing
caused our queries to fail. These failed jobs hung around in our
database but we never did anything with them. We suspect that some of
the users' complaints about publications not reaching Valvas was due to
this. Our first step towards fixing that is to ensure that we retry
failed jobs up to 5 times.

Further steps will be taken to alleviate the process, by making the
status of the export jobs more explicit so that users can track it.

sergiofenoll committed:
chore: log when a job has been retried too many times

ValenberghsSven committed:
implement https for prefix besluitvorming

tdn committed:
removed unnecessary function

tdn committed:
Merge branch 'feature/KAS-3875-prefix-besluitvorming-https' of github.com:kanselarij-vlaanderen/themis-export-service into feature/KAS-3814-merge-app-themis-export

tdn reviewed (approved):
merged the https fix into this and tested. Seems to be working fine

ValenberghsSven committed:
Merge branch 'feature/KAS-3983-retry-public-export-jobs' of github.com:kanselarij-vlaanderen/themis-export-service into feature/KAS-3814-merge-app-themis-export
